src/main/java/ua/shpp/eqbot/command/StartCommand.java

```diff
@@ -0,0 +1,22 @@
+package ua.shpp.eqbot.command;
+
+import org.telegram.telegrambots.meta.api.objects.Update;
+import ua.shpp.eqbot.service.SendBotMessageService;
+
+/**
+ * Start {@link Command}.
+ */
+public class StartCommand implements Command {
```

Comment:
Can I register again on the start menu? For example, if the city or name has changed.

Reply:
class start commander, its implementation is another task